examples/machine-learning/a3-ultragpu-8g/a3ultra-slurm-blueprint.yaml

```diff
@@ -44,6 +43,11 @@ vars:
   nccl_plugin_version: v1.0.5
   base_network_name: $(vars.deployment_name)
 
+  #Provisioning models (set to true or fill in reservation name, pick only one)
+  a3u_reservation_name: "" # supply reservation name
```

Comment:
The fact that you provide `""` value renders this bluprint on-deman by default. Do you agree?